jenkins/catapult/runDockerTests.py

```diff
@@ -55,7 +54,7 @@ def main():
 	parser = argparse.ArgumentParser(description='catapult tests runner')
 	parser.add_argument('--image', help='docker tests image', required=True)
 	parser.add_argument('--compiler-configuration', help='path to compiler configuration yaml', required=True)
-	parser.add_argument('--user', help='docker user', required=True)
+	parser.add_argument('--user', help='docker user')
```

Comment:
not sure, but I'd probably leave it required and let windows build pass some dummy

Reply:
Thanks will update